extract/spine.js

```diff
@@ -28,7 +28,7 @@ export default function spine(xml, tocItem=false) {
   const spine = normalise(
     items.map(({ idref:id, linear, properties}) => ({
       id,
-      linear: linear === 'yes',
+      linear: linear !== undefined ? linear === 'yes' : true,
```

Comment:
typeof check instead of: `linear !== undefined`

however, should we do something like `typeof linear === 'string'` instead?